lib/workload/stateless/stacks/bs-runs-upload-manager/deploy/stacks/bs_runs_upload_manager_stack.ts

```diff
@@ -0,0 +1,74 @@
+import * as cdk from 'aws-cdk-lib';
+import { Construct } from 'constructs';
+import { BsRunsUploadManagerConstruct } from '../constructs/bs_runs_upload_manager_stack';
+import { LambdaLayerConstruct } from '../constructs/lambda_layer';
+import * as secretsManager from 'aws-cdk-lib/aws-secretsmanager';
+import * as events from 'aws-cdk-lib/aws-events';
+import path from 'path';
+
+export interface BsRunsUploadManagerConfig {
+  // Define construct properties here
+  ica_token_secret_id: string; // IcaSecretsPortal
+  portal_token_secret_id: string; // orcabus/token-service-jwt
+  basespace_token_secret_id: string; // /manual/BaseSpaceAccessTokenSecret
+  gds_system_files_path: string; // gds://development/primary_data/temp/bs_runs_upload_tes/
+  eventbus_name: string; // /umccr/orcabus/stateful/eventbridge
+}
+
+export type BsRunsUploadManagerStackProps = BsRunsUploadManagerConfig & cdk.StackProps;
+
+export class BsRunsUploadManagerStack extends cdk.Stack {
+  constructor(scope: Construct, id: string, props: BsRunsUploadManagerStackProps) {
+    super(scope, id, props);
+
+    // Set lambda layer arn object
+    const lambda_layer_obj = new LambdaLayerConstruct(this, 'lambda_layer', {
+      layer_directory: __dirname + '/../../../layers',
```

Comment:
If you happen to refactor folder structure and push this source file up then it will just be 1 level out. `../layers` instead. Same apply down under path fixes.

Reply:
Sorry! Had a bit of a cdk refactor process, will make sure I synth before re-pushing to confirm everything is in the right directory?

Reply:
No sorry need. Understood. All good.! 👍